### PR TITLE
audio+network patch

### PR DESCRIPTION
**Pull request by Noname101-pixel**

### CODE REVIEW

File: src/Game.cpp [hunk 1]
```diff
@@ -59,6 +56,9 @@ void main()
 	p_enemies[1] = new Turret(200.0f, 600.0f, &SH);
 	p_enemies[0] = new Smallboy(600.0f, 500.0f, &SH);
 
+	SH.load_play_audio("laser.wav");
+
+
```
Comment (jeremyhall04) [inline src/Game.cpp:61]: Please clean this whitespace

File: src/Game.cpp [hunk 2]
```diff
@@ -150,24 +151,20 @@ void main()
...
 
-			size = (4 * sizeof(float)) + sizeof(double) + sizeof(bool); //calculating buffer size
-
 			send6(buffer_out, size, IP_address_send, sock, port);
 
 			//Sleep(15);
 
 			//______________________________RECEIVE DATA__________________________________________//
-			for (i = 0; i < 3; i++)
+			for (i = 0; i < 3; i++) // Retry 3 times
```
Comment (jeremyhall04) [inline src/Game.cpp:159]: You don't need the for loop, the while loop checks infinitely

File: src/SceneHandler/SceneHandler.cpp [hunk 3]
```diff
@@ -3,6 +3,15 @@
...
 
+#include <Windows.h>
+#include <MMSystem.h>
+#include <cmath>
+#include <cstring>
+#include <cstdlib>
+#pragma comment(lib,"winmm.lib")
+
+
+
```
Comment (jeremyhall04) [inline src/SceneHandler/SceneHandler.cpp:14]: my guy this whitespace

File: src/Game.cpp (hunk 1 above)
Comment (Noname101-pixel) [reply src/Game.cpp:61]: Never

File: src/SceneHandler/SceneHandler.cpp [hunk 4]
```diff
@@ -61,5 +70,41 @@ void SceneHandler::load_level(int level)
...
+		return;
+	}
+
+	fin.read(audio_buffer, n);
+
+	fin.close();
+
+	PlaySoundA(audio_buffer, NULL, SND_ASYNC | SND_LOOP);
+
+
```
Comment (jeremyhall04) [inline src/SceneHandler/SceneHandler.cpp:109]: More whitespace

**Review by jeremyhall04 — COMMENTED** [review]
Theres a lot of whitespace to clean up.

Also for loop is unnecessary

File: src/Game.cpp (hunk 2 above)
Comment (jeremyhall04) [reply src/Game.cpp:159]: Ok someone explained it to me its all good

**Comment by jeremyhall04** [comment]
This was not ready to push

**Pull request closed without merging**